src/components/PopoverMenu.tsx

```diff
@@ -272,7 +272,7 @@ PopoverMenu.displayName = 'PopoverMenu';
 export default React.memo(
     PopoverMenu,
     (prevProps, nextProps) =>
-        !lodashIsEqual(prevProps.menuItems, nextProps.menuItems) &&
```

Comment:
@roryabraham In the past we didn't re-render if the `menuItems` is changed so I think it's fine if we only compare the length of `menuItems` here.

Or If we agree with more re-rendering, I will add the previous change back.

Reply:
> In the past we didn't re-render if the menuItems is changed

What makes you say that? It looks like it was using a deep equality comparison in the `React.memo`?

Reply:
@roryabraham Because we used `!lodashIsEqual(prevProps.menuItems, nextProps.menuItems)` not `lodashIsEqual(prevProps.menuItems, nextProps.menuItems)`

Reply:
Pretty sure `!lodashIsEqual` was a bug before, since the 2nd arg to `React.memo` should return true if the props _are_ equal